server/service/FlightService.js

```diff
@@ -100,7 +100,7 @@ const flightService = {
 
     if (
       today.getHours() + today.getTimezoneOffset() / 2 >=
-      SWITCHOVER_HOUR_TODAY_RANKING
+      SWITCHOVER_HOUR_TODAY_RANKING - 1 // Was off by one
```

Comment:
Äh... ich glaube das ursprüngliche Problem liegt in Zeile 102 und der Reihenfolge von Addition und Devision